OMFAPI/Program.cs

```diff
@@ -21,7 +21,8 @@ public static class Program
         // Constant for determining the pause between sending OMF data messages
         private const int SendSleep = 1000;
 
-        private static readonly HttpClient _client = new ();
+        private static readonly List<HttpClient> _clients = new ();
```

Comment:
What do you think of just having the client be part of the ```Endpoint``` object? I think it could simply things.

Reply:
That's probably better, but my only reservation is that the test rebuilds the endpoints. So the logic for the cert handler would have to live in both places (not doing that could cause the cert check to fail for the cleanup messages). As it is now, both the test and the main program should be pretty robust in each endpoint knowing its index, and with the List<HttpClient> object staying alive until the whole thing is done, it should safely access the client that it needs each time, implicitly picking up the cert handler callback.

Reply:
i like it better on the object too even if that means rebuilding some of the code in the test, because the addition of having to handle the httpclient in the main part of the code feels odd to me.  I would rather have that all handled by the endpoint object.